pages/common/flake8.md

```diff
@@ -0,0 +1,20 @@
+# flake8
+
+> Tool to check the style and quality of Python code.
+> More information: <https://flake8.pycqa.org/>.
+
+- Lint a file or directory:
+
+`flake8 {{path/to/file_or_directory}}`
+
+- Show source code for each error:
+
+`flake8 --show-source {{path/to/file_or_directory}}`
+
+- Ignore a list of rules:
+
+`flake8 --ignore {{rule1,rule2}} {{path/to/file_or_directory}}`
+
+- Exclude files matching the given patterns:
```

Comment:
Is there maybe a command to list all available rules and patterns?

Reply:
I can't find one to list the rules. There is a list at https://www.flake8rules.com/.
As file patterns go, flake8 uses globs and substring matches.

Reply:
Then maybe we should be more specific here:
```suggestion
- Exclude files matching the given globs or substrings:
```
And for the rules, how about
```diff
- Ignore a list of rules:
+ Ignore a list of rules. All available rules can be found at flake8rules.com:
```

Reply:
Terrific! Will push changes in next commit.